Assets/Scripts/Managers/AudioManager.cs

```diff
@@ -0,0 +1,78 @@
+using UnityEngine.Audio;
+using System;
+using UnityEngine;
+
+
+public class AudioManager : MonoBehaviour
+{
+    public static AudioManager Instance;
+    [SerializeField] public AudioMixerGroup musicMixerGroup;
+    [SerializeField] public AudioMixerGroup sfxMixerGroup;
+    [SerializeField] public Sound[] sounds;
```

Comment:
No need for both `[SerializeField]` and `public` - if the field is `public`, it's already serialized in the inspector